ipfs-cluster/ipfs-cluster-deployment.yml

```diff
@@ -1,77 +0,0 @@
-apiVersion: extensions/v1beta1
```

Comment:
Hmm this diff is confusing me.  I didn't delete this and after checking the files manually I see that there is no difference between this file on the master branch and my incoming branch.  Both apiVersion lines are present in the deployment on my branch